docs/managing_deis/running-deis-without-ceph.rst

```diff
@@ -35,15 +35,15 @@ the removal of Ceph. The necessary changes are described below.
 Logger
 ^^^^^^
 
-The :ref:`logger` component provides a syslog-compatible endpoint to consume
-application logs, which it writes to a shared Ceph filesystem. These logs are
-read by the :ref:`controller` component. The :ref:`logspout` talks to the Docker
-daemon on each host, listens for log events from running applications, and ships
-them to the logger.
+The :ref:`logspout` component attaches to Docker containers on each host and
+listens for log events from platform components and running applications.  It
+ships these to the :ref:`logger` component.  By default, the logger writes the
+logs to a distributed Ceph filesystem. These logs can then be fetched by the
+:ref:`controller` component via HTTP.
 
-The Logger component is not necessary in a Ceph-less Deis cluster. Instead of
-using the Logger, we will route all the logs directly to another syslog
-compatible endpoint.
+In a Ceph-less clutser, the Logger component should be configured, instead, to
+use in-memory log storage.  Optionally, a drain may also be configured to forward
```

Comment:
Ah yes, I'm being a little dumb here. Of course you'd want to use in-memory log storage when Ceph doesn't exist. Disregard!